src/main/java/hudson/plugins/git/UserMergeOptions.java

```diff
@@ -21,16 +21,18 @@
     private String mergeRemote;
     private String mergeTarget;
     private String mergeStrategy;
+    private String fastForwardMode;
 
     @DataBoundConstructor
-    public UserMergeOptions(String mergeRemote, String mergeTarget, String mergeStrategy) {
+    public UserMergeOptions(String mergeRemote, String mergeTarget, String mergeStrategy, String fastForwardMode) {
```

Comment:
I do not know how to code a "new constructor with default". I need to find an example. Any pointer appreciated.